operate/qa/integration-tests/src/test/java/io/camunda/operate/webapp/security/es/ElasticSearchUserDetailsServiceIT.java

```diff
@@ -9,16 +9,18 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import io.camunda.authentication.entity.CamundaUser;
 import io.camunda.operate.property.OperateProperties;
 import io.camunda.operate.schema.indices.UserIndex;
 import io.camunda.operate.util.OperateAbstractIT;
 import io.camunda.operate.util.SearchTestRule;
 import io.camunda.operate.util.TestApplication;
 import io.camunda.operate.util.searchrepository.TestSearchRepository;
 import io.camunda.operate.webapp.security.auth.OperateUserDetailsService;
-import io.camunda.operate.webapp.security.auth.User;
+import io.camunda.operate.webapp.security.auth.Role;
 import java.io.IOException;
 import java.util.Map;
+import org.elasticsearch.core.List;
```

Comment:
use `java.util.List` and remove the dependency to `org.elasticsearch:elasticsearch-core`